src/languages/db2/db2.formatter.ts

```diff
@@ -185,7 +185,7 @@ export const db2: DialectOptions = {
     reservedKeywords: keywords,
     reservedFunctionNames: functions,
     stringTypes: [
-      { quote: "''-qq", prefixes: ['G', 'N', 'U&'] },
+      { quote: "''-qq", prefixes: ['G', 'N', 'U'] },
```

Comment:
Is this change of intentional?

Reply:
That is, you seem to be suggesting that `U&''` strings are not supported in DB2 LUW-variant, but instead `U''` strings are supported.

Reply:
Sorry, a mistake from my part - my rebase didn't go as expected. I'll remove this change, since you've already fixed the `U&''` issue.